django/forms/widgets.py

```diff
@@ -875,10 +875,10 @@ class SplitDateTimeWidget(MultiWidget):
     supports_microseconds = False
     template_name = 'django/forms/widgets/splitdatetime.html'
 
-    def __init__(self, attrs=None, date_format=None, time_format=None):
+    def __init__(self, attrs=None, date_format=None, time_format=None, date_attrs=None, time_attrs=None):
```

Comment:
You also need to update `SplitHiddenDateTimeWidget.__init__()` to add and pass `date_attrs` and `time_attrs` so that attributes can be correctly altered for that widget too. You mentioned this widget in the ticket but didn't make the changes in this PR.